extensions-core/lookups-cached-global/src/main/java/io/druid/server/lookup/namespace/JdbcCacheGenerator.java

```diff
@@ -132,6 +127,30 @@
     }
   }
 
+  private String buildLookupQuery(String table, String filter, String keyColumn, String valueColumn)
+  {
+    String query;
```

Comment:
This variable is not needed, there could be two return statements in `if` and `else` blocks